src/core/generator/tests/json_to_config_spec.rs

```diff
@@ -15,7 +17,10 @@ struct JsonFixture {
 datatest_stable::harness!(
     run_json_to_config_spec,
     "src/core/generator/tests/fixtures/json",
-    r"^.*\.json"
+    r"^.*\.json",
+    open_api_to_config_spec::run_open_api_to_config_spec,
+    "src/core/generator/tests/fixtures/openapi",
+    r"^.*\.yml"
 );
```

Comment:
Move these tests to `open_api_to_config_spec.rs`

Reply:
created a new file named `config_gen.rs` which makes call to both `run_json_to_config_spec` and `run_openapi_to_config_spec`